api/src/anipy_api/provider/providers/gogo_provider.py

```diff
@@ -387,3 +369,20 @@ def _get_episode_ajax(
         ep_list.reverse()
 
         return ep_list
+
+    @functools.lru_cache()
```

Comment:
I deliberately decoupled this so the cache is not bound to the class instance, this should have been   commented, sorry.

Reply:
That's actually fair. I'll move it back out and I'll write a comment